_includes/youtube-embed.html

```diff
@@ -0,0 +1,3 @@
+<div class="youtube">
```

Comment:
It feels like these changes might not belong to this PR

Reply:
Yes, I will get rid of it.